pkg/cache/context_test.go

```diff
@@ -49,16 +49,46 @@ import (
 )
 
 const (
-	Host1  = "HOST1"
-	appID1 = "app00001"
-	appID2 = "app00002"
-	appID3 = "app00003"
-)
+	// NOTE: why this variable is public of package
```

Comment:
I don't think it should be public, but the name `host1` will be duplicate to this variable:
https://github.com/apache/yunikorn-k8shim/blob/741c0d801ac4530669b8850706efe3f0bc0d5718/pkg/cache/context_test.go#L610-L611

So, a rename is needed.